src/docs/BUILD.md

```diff
@@ -10,20 +13,16 @@
 
 ### Build Instructions
 
-1. Clone OpenVINO GenAI repository and init submodules:
+1. Build OpenVINO from sources following the [instructions](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md)
+2. Install OpenVINO following the [instructions](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/installing.md)
+3. Clone OpenVINO GenAI repository and init submodules:
     ```sh
     git clone --recursive https://github.com/openvinotoolkit/openvino.genai.git
     cd openvino.genai
     ```
-2. Download OpenVINO archive and install dependencies:
-    ```sh
-    mkdir ./ov/
-    curl https://storage.openvinotoolkit.org/repositories/openvino/packages/pre-release/2024.3.0rc1/linux/l_openvino_toolkit_ubuntu20_2024.3.0.dev20240711_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
-    sudo ./ov/install_dependencies/install_openvino_dependencies.sh
-    ```
-3. Build the project:
+4. Build the project:
     ```sh
-    source ./ov/setupvars.sh
+    source <OPENIVNO_INSTALL_DIR>/setupvars.sh
```

Comment:
installing.md uses `INSTALLDIR`, which is strange, because the docs use `INSTALL_DIR` https://docs.openvino.ai/2024/get-started/install-openvino/install-openvino-archive-windows.html. Pick one of them them and update two other locations: here and one of ov.

Reply:
Updated command to use `INSTALL_DIR `

Reply:
PR to docs: https://github.com/openvinotoolkit/openvino/pull/25678